app/containers/ItunesContainer/index.tsx

```diff
@@ -34,10 +35,33 @@ const CustomInput = styled(Input)`
 `;
 
 const ItunesContainer = ({ dispatchArtistName, songData, loading, songListError }: ItuneContainerProps) => {
+  const [paginationParams, setPaginationParams] = useState({ pageNumber: 1, pageSize: 10 });
```

Comment:
please check https://reactrouter.com/en/main/hooks/use-search-params

Reply:
i don't think we've got to use an internal state here if we can use useSearchParams